data_diff/tracking.py

```diff
@@ -0,0 +1,114 @@
+#
+# This module contains all the functionality related to the anonymous tracking of data-diff use.
+#
+
+import toml
+import logging
+import os
+import json
+import platform
+from time import time
+from typing import Any, Dict, Optional
+import urllib.request
+from uuid import uuid4
+
+TRACK_URL = "https://api.perfalytics.com/track"
+START_EVENT = "os_diff_run_start"
+END_EVENT = "os_diff_run_end"
+TOKEN = "ccb8c3a6-3b6f-445c-ad67-994efa7bd020"
+
+DEFAULT_PROFILE = os.path.expanduser("~/.datadiff.toml")
+
+
+def _load_profile():
+    try:
+        with open(DEFAULT_PROFILE) as f:
+            conf = toml.load(f)
+    except FileNotFoundError as e:
+        conf = {}
+
+    if "anonymous_id" not in conf:
+        conf["anonymous_id"] = str(uuid4())
+        with open(DEFAULT_PROFILE, "w") as f:
+            toml.dump(conf, f)
+    return conf
+
+
+g_tracking_enabled = True
+g_anonymous_id = None
+
+
+def disable_tracking():
+    global g_tracking_enabled
+    g_tracking_enabled = False
+
+
+def is_tracking_enabled():
+    return g_tracking_enabled
+
+
+def get_anonymous_id():
+    global g_anonymous_id
+    if g_anonymous_id is None:
+        profile = _load_profile()
+        g_anonymous_id = profile["anonymous_id"]
+    return g_anonymous_id
+
+
+def create_start_event_json(diff_options: Dict[str, Any]):
+    return {
+        "event": "os_diff_run_start",
+        "properties": {
+            "distinct_id": get_anonymous_id(),
+            "token": TOKEN,
+            "time": time(),
```

Comment:
Contrary to what I said about `time.monotonic()` above, this is a good place for `time.time()` — i.e. the UTC moment when it was executed.